### ISSUE TITLE
AzureServiceBus: new feature to show Dead letter

### ISSUE THREAD

**Issue by adelinor** [jmstoolbox/jmstoolbox#138]
## Specifications
- JMSToolBox version: 5.14.0
- OS (Name, version, 32/64bits): Windows 10 Enterprise 64 bits
- Q provider (kind, version) : AzureServiceBus

## Expected Behavior
AzureServiceBus creates a Dead letter queue behind the scenes. It would be very helpful if this implicitly created queue could be seen from JMSToolBox.

## Actual Behavior
JMSToolBox shows all queues that are explicitly created.

## Steps to Reproduce the Problem

  1. Open a session configured for AzureServiceBus
  1. View existing queues under the session's name

## Logs
Not applicable

**Comment by adelinor** [comment]
Dear @titou10titou10 ,

It seems that a dead letter queue gets created for every queue. According to the article, https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues, the naming convention is `<queue name>/$deadletterqueue`

Kind regards

**Comment by titou10titou10** [comment]
OK

 - Are those DLQ accessible/visible from a JMS client? Are they considered "first class" destinations from a JMS perspective? 
 - How to retrieve the list of those DLQ from a JMS client?  As said before. the only proprietary API I know to get the list of destination does not do that (ie`com.microsoft.azure.servicebus.management.ManagementClient`)

**Comment by adelinor** [comment]
Yes as per the unit test code added as a comment on issue #137 , when it exists, the dead letter is accessible from the JMS api as any other queue.
The [isEnableDeadLetteringOnMessageExpiration()](https://docs.microsoft.com/en-us/java/api/com.microsoft.azure.servicebus.management.queuedescription.isenabledeadletteringonmessageexpiration?view=azure-java-stable#com-microsoft-azure-servicebus-management-queuedescription-isenabledeadletteringonmessageexpiration()) method would tell if the dead letter is configured for a given queue.

How to retrieve the list of those DLQ from a JMS client?
When retrieving the list of queues with `com.microsoft.azure.servicebus.management.ManagementClient`, then using the `QueueDescription` you can find out if a corresponding DLQ exists for a given queue.
